bin/gxa-index-check-experiments.sh

```diff
@@ -14,7 +14,7 @@ if [[ "$status" -ne "0" ]]; then
     echo $out
     exit 1
 fi 
-for exp in $ATLAS_EXPS; do
+for exp in $EXPERIMENT_ID; do
```

Comment:
If this is holding various experiments, you might want to reflect that in the variable name for clarity.
```suggestion
for exp in $EXPERIMENT_IDS; do
```

Reply:
@pcm32 The thing is that it might be either multiple experiments or a single one depending on a use case, so I think it's better to leave it as EXPERIMENT_ID, which can mean both